docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -156,8 +156,8 @@ You can add (+), subtract (-), and multiply (*) with this simple calculator.
 
 You can try a few of the sample calculator functions:
 
-* `http://127.0.0.1:<port>/starter/calculator/{op}/{a}/{b}`, where you can input one of the available operations `(+, _, *)`, and an integer a, and an integer b.
-* So for `http://127.0.0.1:<port>/starter/calculator/+/10/3` you see: `10+3=13`.
+* `http://127.0.0.1:<port>/starter/calculator/{op}/{a}/{b}`, where you can input one of the available operations **(+, _, *)**, and an integer a, and an integer b.
+* So for `http://127.0.0.1:<port>/starter/calculator/+/10/3` you see: **10+3=13**.
```

Comment:
I'm guessing "(+, _, *)" and "10+3=13" aren't UI elements. They could be left in code phrase format.